css/helpers/flexbox.css

```diff
@@ -12,17 +12,30 @@
   flex-wrap: wrap;
 }
 
-.flex-center{
+.flex-center {
   justify-content: center;
   align-items: center;
 }
 
-
 /* Justify */
-.justify-sb{
+.justify-sb {
   justify-content: space-between;
 }
```

Comment:
It's better to  use margin-left : auto;

Reply:
It is a helper-class specific for flexbox.